program_xdp.go

```diff
@@ -70,21 +70,17 @@ func (t XdpResult) String() string {
 
 // XDP eBPF program (implements Program interface)
 type xdpProgram struct {
-	BaseProgram
+	*BaseProgram
```

Comment:
Hm, why it is pointer?

Reply:
I changed this to be passed by pointer as it can be quite a large struct. If you'd like to keep it by value then I can change it back, there's no real performance concern since it's only executed once per program creation but I'm just used to passing by pointer for large structs.